js/ad_manager_vast.js

```diff
@@ -316,7 +316,8 @@ OO.Ads.manager(function(_, $) {
      * @returns {string} The Vast version.
      */
     var getVastVersion = _.bind(function(vastXML) {
-      return $(vastXML.firstChild).attr('version');
+      var vastTag = $(vastXML).find("VAST")[0];
```

Comment:
Related to the comment for isValidRootTagName, maybe we can introduce a getVastRoot function to retrieve the root of the Vast Tag and use it when we need to reference the root, like for validation. What do you think?